src/main/java/org/jenkinsci/plugins/build_view_column/BuildViewColumnDescriptor.java

```diff
@@ -16,6 +16,10 @@ protected BuildViewColumnDescriptor() {
     }
 
     public static DescriptorExtensionList<BuildViewColumn,BuildViewColumnDescriptor> all() {
-        return Jenkins.getInstance().getDescriptorList(BuildViewColumn.class);
+        try{
+            return Jenkins.getInstance().getDescriptorList(BuildViewColumn.class);
+        } catch(NullPointerException ex){
```

Comment:
:ant: Do not use this idiom. For older versions of Jenkins, use

```java
Jenkins j = Jenkins.getInstance();
if (j == null) {
    throw new IllegalStateException();
}
return j.getDescriptorList(BuildViewColumn.class);
```

In newer versions, you can use `Jenkins.getActiveInstance()`; in still newer versions, simply `Jenkins.getInstance()` passes FindBugs.

But best is to use `ExtensionList.lookup(BuildViewColumn.class)`. It returns a more generic type but that is unlikely to matter.